io/plugins/eu.esdihumboldt.hale.io.gml.geometry/src/eu/esdihumboldt/hale/io/gml/geometry/handler/PolygonHandler.java

```diff
@@ -160,10 +162,11 @@ public Object createGeometry(Instance instance, int srsDimension, IOProvider rea
 						// GeometryProperty<LinearRing> instance
 						GeometryProperty<LinearRing> ring = (GeometryProperty<LinearRing>) ((Instance) value)
 								.getValue();
-						outerRing.add(ring.getGeometry());
+						outer = ring.getGeometry();
```

Comment:
Before I was adding each value to list but I think we need a last value, while iterating each value.